src/VersionControl/GitVersionControl.php

```diff
@@ -42,7 +42,7 @@ public function getStagedFiles()
         foreach ($this->getFiles() as $file) {
             list($status, $relativePath) = explode("\t", $file);
 
-            $fullPath = $base . DIRECTORY_SEPARATOR . $relativePath;
+            $fullPath = trim($base . DIRECTORY_SEPARATOR . $relativePath);
```

Comment:
> trim may be described as rtirm+ltrim, and here you don't need ltrim, so why to waste time on doing it?

Thanks, that's a much more helpful comment, but I'll reiterate my statement about this micro-optimisation not being a concern enough to hold up the merge. Like I said though, I'd happily review a new PR for the change.